src/Symfony/Component/HttpFoundation/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ CHANGELOG
    according to [RFC 8674](https://tools.ietf.org/html/rfc8674)
  * made the Mime component an optional dependency
  * added `MarshallingSessionHandler`, `IdentityMarshaller`
+ * made `Session` and `SessionBagProxy` calling callback on session usage
```

Comment:
SessionBagProxy is internal

made `Session` accept a callback reporting when the session is being used